### DISCUSSION TITLE
Quarto Projects and Secrets Management

### DISCUSSION THREAD

**Discussion by corvidfox** [brad-cannell/bradcannell_blogdown#18] [Ideas]
I've been playing around with Quarto notebooks, and one thing that I've learned is that Quarto Projects can perform secrets control through a `_environment.local` file, which is kept out of version-control in `.gitignore`. This may have some security and cross-OS or cross-language benefits. There is a learning curve to setting up these project environments, but it can yield consistency similar to an `.Rproj` file.

https://quarto.org/docs/projects/environment.html#managing-secrets

**Comment by mbcann01** [comment]
Sorry, @corvidfox ! I'm just now seeing this. I'm still getting used to people actually posting things on the discussion board as I asked 😂. Thank you for sharing this info!
